src/components/presentational/Layout/Layout.tsx

```diff
@@ -32,31 +33,32 @@ export default function (props: LayoutProps) {
 		className += " " + props.className;
 	}
 
-	if (props.bodyBackgroundColor) {
-		document.body.style.backgroundColor = props.bodyBackgroundColor;
+	let colorScheme: "light" | "dark" = "light";
```

Comment:
What is the intent of the second portion of or? dark = light is false. 
let colorScheme: "light" | false;
Is the intent to default to light, bc resolveColor takes light|dark?

Reply:
yeah "light" | "dark" is the type...that is typescript syntax